test/e2e/framework/util.go

```diff
@@ -413,6 +413,7 @@ func CheckTestingNSDeletedExcept(ctx context.Context, c clientset.Interface, ski
 // WaitForServiceEndpointsNum waits until there are EndpointSlices for serviceName
 // containing a total of expectNum endpoints. (If the service is dual-stack, expectNum
 // must count the endpoints of both IP families.)
+// Deprecated: use e2eendpointslice.WaitForEndpointCount or other related functions.
```

Comment:
I'm happy to go find the other places that `WaitForServiceEndpointsNum` is used and switch them over to the new functions that you made, after this PR lands (if that provides value?)

Reply:
Yes (and there are probably places that could be using the other new helpers too). I just didn't want to poke at other people's tests in this PR.